plugins/woocommerce-blocks/packages/components/store-notices-container/index.tsx

```diff
@@ -2,10 +2,7 @@
  * External dependencies
  */
 import { useSelect, useDispatch } from '@wordpress/data';
-import {
-	PAYMENT_STORE_KEY,
-	STORE_NOTICES_STORE_KEY,
-} from '@woocommerce/block-data';
+import { PAYMENT_STORE_KEY, storeNoticesStore } from '@woocommerce/block-data';
```

Comment:
There's a select in this file that still uses `PAYMENT_STORE_KEY` instead of `storeNoticesStore` descriptor